### PR TITLE
feat: Stacks improvements

### PR DESCRIPTION
**Pull request by maciaszczykm**
## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.

### CODE REVIEW

**Review by michaeljguarino — APPROVED** [review]
why did the docker image build fail?

**Comment by maciaszczykm** [comment]
```
63.00 src/components/account/audits/AuditsList.tsx(173,58): error TS2345: Argument of type 'EventTarget' is not assignable to parameter of type 'HTMLDivElement'.
```

Checking it now. It's file that I did not change.

**Comment by jsladerman** [comment]
> ```
> 63.00 src/components/account/audits/AuditsList.tsx(173,58): error TS2345: Argument of type 'EventTarget' is not assignable to parameter of type 'HTMLDivElement'.
> ```
> 
> Checking it now. It's file that I did not change.

Good chance it was caused by the lockfile update

File: assets/src/components/cd/pipelines/utils/nodeLayouter.tsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ function measureNode(node: FlowNode, zoom) {
   let domNode
 
   try {
-    const selector = `[data-id="${node.id}"]`
-
-    domNode = document.querySelector(selector)
+    domNode = document.querySelector(CSS.escape(`[data-id="${node.id}"]`))
```
Comment (jsladerman) [inline assets/src/components/cd/pipelines/utils/nodeLayouter.tsx:8]: seems to be causing a weird bug with the layout

**Pull request merged**